modules/ROOT/pages/clustering/disaster-recovery.adoc

```diff
@@ -173,7 +175,7 @@ SHOW SERVERS;
 ==== Path to correct state
 Use the following steps to remove lost servers and add new ones to the cluster.
 To remove lost servers, any allocations they were hosting must be moved to available servers in the cluster.
-This can be done in two different ways:
+This is done in two different ways:
```

Comment:
```suggestion
This is done in two different steps:
```

Reply:
I also like the suggestion with "steps".